publiq-ui/pub-alert.stories.mdx

```diff
@@ -0,0 +1,76 @@
+import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs/blocks';
+import PubAlert, { AlertVariant } from './pub-alert.vue';
+
+<Meta title="pub-alert" component={PubAlert} />
```

Comment:
Remember to put this in the `Components` group now since the PR with the grouping was merged.
So:
```suggestion
<Meta title="Components/pub-alert" component={PubAlert} />
```